prog/vm/github_runner.rb

```diff
@@ -31,11 +31,12 @@ def self.assemble(installation, repository_name:, label:, default_branch: nil)
   end
 
   def pick_vm
+    location = project.get_ff_runner_preferred_location || label_data["location"]
     skip_sync = true
     pool = VmPool.where(
       vm_size: label_data["vm_size"],
       boot_image: label_data["boot_image"],
-      location: label_data["location"],
+      location: location,
```

Comment:
if we do this for more customers it might become a problem that we have to pin to `hetzner-fsn1` even though we probably would want to pin to 'github-runners' in fns1? maybe not now but in the future we could introduce an additional filter for data_center and keep 'github-runners' as location for this case but with an additional 'FSN1-%' filter for data center.